lib/dnsimple/client/registrar.rb

```diff
@@ -21,6 +21,14 @@ def register(account_id, domain_name, attributes = {}, options = {})
         Dnsimple::Response.new(response, Struct::Domain.new(response["data"]))
       end
 
+
+      def check(account_id, domain_name, attributes = {}, options = {})
+        options  = options.merge(attributes)
+        response = client.get(Client.versioned("/%s/registrar/domains/%s/availability" % [account_id, domain_name]), options)
```

Comment:
Yes. At least in https://github.com/aetrion/dnsimple/blob/master/app/api/api/v2.raml#L126. I think I saw it somewhere else. I will add it here if I find it.